### ISSUE TITLE
Minimize Window ,can pause

### ISSUE THREAD

**Issue by TOGOCHEESE** [Genymobile/scrcpy#2770]
1.17 when Minimize Window ,scrcpy can pause
1.19 can't

I want to be an option

**Comment by rom1v** [comment]
Scrcpy never pause. Even in v1.17. What do you mean by "pause" ?

**Comment by TOGOCHEESE** [comment]
> Scrcpy never pauses. Even in v1.17. What do you mean by "pause" ?

Oh ok
when I use Open Broadcaster Software catch window screen

**Comment by rom1v** [comment]
Probably a difference in SDL version. Try replacing SDL2.dll from the scrcpy release by the one from an older release.

**Comment by TOGOCHEESE** [comment]
> Probably a difference in SDL version. Try replacing SDL2.dll from the scrcpy release by the one from an older release.

Can't be an available option? :)

**Comment by rom1v** [comment]
> Can't be an available option? :)

No, SDL is a dependency of scrcpy, if its behavior when the window is minimized has changed (maybe "has been fixed"), then scrcpy could not change the behavior.
